ChopChop/Storage/FirebaseDatabase.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  FirebaseDatabase.swift
+//  ChopChop
+//
+//  Created by Cao Wenjie on 27/3/21.
+//
+import Firebase
+//import FirebaseFirestoreSwift
+import Combine
+import CombineFirebase
+
+struct FirebaseDatabase {
+    private let recipePath: String = "recipes"
+    private let userPath: String = "users"
+    private let db = Firestore.firestore()
+
+    func addRecipe(recipe: OnlineRecipeRecord) throws -> String {
+        do {
+            return try db.collection(recipePath).addDocument(from: recipe).documentID
+        } catch {
+            throw FirebaseError.addRecipeError(message: "Unable to add recipe: \(error.localizedDescription)")
+        }
+    }
+    func removeRecipe(recipeId: String) throws {
```

Comment:
need to update local database's recipe also

Reply:
This will be done in storage manager, I have updated it.